src/Widgets/AppContainers/InstalledPackageRowGrid.vala

```diff
@@ -147,26 +146,20 @@ public class AppCenter.Widgets.InstalledPackageRowGrid : AbstractPackageRowGrid
                 xalign = 0
             };
 
-            var dialog_close_button = new Gtk.Button.with_label (_("Close")) {
-                halign = Gtk.Align.END,
-                valign = Gtk.Align.END,
-                vexpand = true,
-                margin_top = 12
-            };
-
             var releases_dialog_box = new Gtk.Box (Gtk.Orientation.VERTICAL, 12) {
-                margin_top = 12,
                 margin_end = 12,
-                margin_bottom = 12,
-                margin_start = 12
+                margin_start = 12,
+                vexpand = true
             };
             releases_dialog_box.add (releases_title);
             releases_dialog_box.add (release_description);
-            releases_dialog_box.add (dialog_close_button);
+            releases_dialog_box.show_all ();
```

Comment:
Does Box really still use show_all instead of present?

Reply:
Yeah because this branch is still gtk3